theme/less/components/forms.less

```diff
@@ -0,0 +1,110 @@
+/*
+---
+name: Forms
+category: Components
+---
+
+Simple forms for simple needs.
+
+We're assuming some well formed `fieldsets` with proper `legends`, this is mostyly used in the login/signup flows.
+
+A well formed form (eh) should include one main fieldset with a proper legend.
+
+```form.html
+    <form class="form"">
+        <fieldset>
+            <legend>Legend</legend>
+            <div class="field">
+                <label for="field" class="required">
+                    Label
+                </label>
+                <input id="field" name="field" type="email" required></input>
+            </div>
+        </fieldset>
+    </form>
+```
+*/
+
+.form-container {
+    background-color: @white;
+    border: 1px solid @grey-100;
+    border-radius: 4px;
+    padding: @spacing-lg;
+}
+
+form.form {
+    padding: @spacing-sm 0;
+
+    legend {
+        padding: 0;
+    }
+
+    fieldset {
+        border-radius: 4px;
+        margin-inline-start: 0;
+        margin-inline-end: 0;
+        padding-block-start: 0;
+        padding-block-end: 0;
+        padding-inline-start: 0;
+        padding-inline-end: 0;
+        min-inline-size: min-content;
```

Comment:
https://caniuse.com/?search=min-inline-size

What does it look like on IE 11 ?